lib/timescaledb/cleaner.ex

```diff
@@ -1,7 +1,12 @@
 defmodule Membrane.RTC.Engine.TimescaleDB.Cleaner do
   @moduledoc """
-  Worker responsible for inserting data to database and deleting data from it.
+  Worker responsible for deleting obsolete records from the database.
   By default started in project application module in supervison tree with params passed in application config.
```

Comment:
```suggestion
  By default started under the application's supervision tree with params passed in the application config.
```

One more thing is missing here I think: docs for this config - what keys and for which application have to be set. Putting the part of the config would do the trick.